react/components/EXPERIMENTAL_Table/BulkActions.tsx

```diff
@@ -51,30 +50,28 @@ const BulkActions: FC<BulkActionsProps> = ({
         overflow: hasRowsSelected ? 'auto' : 'hidden',
         transition: BULK_ACTIONS_TRANSITION,
       }}>
-      {hasBulkActions && (
-        <div className="flex flex-row">
-          {hasPrimaryBulkAction && (
-            <div className="mr4">
-              <Button
-                variation="secondary"
-                size="small"
-                onClick={() => main.onClick(bulkActionsReturnedParameters)}>
-                {main.label}
-              </Button>
-            </div>
-          )}
-          {hasSecondaryBulkActions && (
-            <ActionMenu
-              label={texts.secondaryActionsLabel}
-              buttonProps={{ variation: 'secondary', size: 'small' }}
-              options={others.map(el => ({
-                label: el.label,
-                onClick: () => el.onClick(bulkActionsReturnedParameters),
-              }))}
-            />
-          )}
-        </div>
-      )}
+      <div className="flex flex-row">
+        {hasPrimaryBulkAction && (
+          <div className="mr4">
+            <Button
+              variation="secondary"
+              size="small"
+              onClick={() => main.onClick(bulkActionsReturnedParameters)}>
+              {main.label}
+            </Button>
+          </div>
+        )}
+        {hasSecondaryBulkActions && (
+          <ActionMenu
+            label={texts.secondaryActionsLabel}
+            buttonProps={{ variation: 'secondary', size: 'small' }}
+            options={others.map(el => ({
+              label: el.label,
+              onClick: () => el.onClick(bulkActionsReturnedParameters),
+            }))}
+          />
+        )}
+      </div>
```

Comment:
I think this change is unnecessary, actually... @matheusps has an opinion about that?